crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -40,19 +40,26 @@ pub struct Diagnostic {
 }
 
 impl Diagnostic {
+    /// Add a new value measured now.
     pub fn add_measurement(&mut self, value: f64) {
```

Comment:
I am really not thrilled with the fact that all measurements are assumed to be f64, but that's probably out of scope.

Reply:
yup me neither 🙁